src/app/menu/__tests__/api.test.ts

```diff
@@ -0,0 +1,33 @@
+import { Menu } from "@/models"
+import { getMenuById } from "../api"
+import { enableFetchMocks, MockParams } from "jest-fetch-mock"
+
+enableFetchMocks()
+
+beforeEach(() => { 
+    fetchMock.resetMocks()
+})
+const mockParams: MockParams = {
+    headers: { "content-type": "application/json" }
+}
+describe("Menu", () => {
+    it("Gets menu by id", async () => {
+        const id = "1"
+        const date: Date = new Date()
+    
+        const mockResponce: Menu = {
+            isActive: true,
+            name: "mock menu",
+            id: "1",
```

Comment:
Should change id from a hardcoded value to the id variable @ line 15

Reply:
omg ty bestie